x509/extensions.go

```diff
@@ -120,6 +120,19 @@ func (cp *CertificatePoliciesData) MarshalJSON() ([]byte, error) {
 	return json.Marshal(policies)
 }
 
+// GeneralNames corresponds an X.509 GeneralName defined in
+// Section 4.2.1.6 of RFC 5280.
+//
+// GeneralName ::= CHOICE {
```

Comment:
That's not Go code, it's a comment with documentation copy/pasted from the RFC? That's ASN1 notation :)